src/pivotal-ui/components/lists/lists.scss

```diff
@@ -155,8 +158,11 @@ parent: list
 
 */
 
+
+
 .list-unordered {
   padding-left: 0;
+  list-style-type: disc;
```

Comment:
I thought this was the default for most browsers... but maybe I'm remembering wrongly.